powerline/listers/i3wm.py

```diff
@@ -0,0 +1,80 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+import re
+
+from powerline.theme import requires_segment_info
+from powerline.lib.shell import run_cmd
+from powerline.lib.dict import updated
+
+
+conn = None
```

Comment:
but should i3-specific `conn` handling and  wm-agnostic `xrandr´ handling go in the same file? I guess it doesn't matter so much when the connection isn't made if it isn't used but still